### DISCUSSION TITLE
Website source code

### DISCUSSION THREAD

**Discussion by CONIGUERO** [Infisical/infisical#137] [Q&A] [answered]
Hi!

I love the project. Very excited and hopeful for its future!

Could the main website and docs sources (apart from the docs markdown itself) be open source? I just love how it's designed and how it flows so beautifully. Would love to take a peek at how it's done.

Thanks!

**Comment by vmatsiiako** [comment, marked as answer]
Hi there! Thank you so much for your kind words :)

For docs, we are actually using Mintlify (you can check it out here: https://github.com/mintlify/mint)

Our landing page is currently not open source because the code is a bit dirty :D It's basically using next.js + tailwind.
I will work on cleaning it up, and will update you here when it's ready.
